FNML/ontology.md

```diff
@@ -57,15 +54,20 @@ or go (so `rr:IRI` by default).
 </p>
 
 <p class="issue" data-number="7" data-format="markdown">
-It is currently unspecified how to override the termtype of a <a>function map</a> result.
+It is currently unspecified how to override the termtype of a [=function term map=] result.
 </p>
 
-### fnml:functionValue
+<p class="issue" data-number="12" data-format="markdown">
+A proper Term map definition in RML is pending.
+For now, we refer to the R2RML spec, but it is assumed these references will be updated based on the evolution of RML.
+</p>
 
-<dfn>fnml:functionValue</dfn> connects the knowledge graph generating triples map with a function execution triples map.
-It has range <a>fnml:FunctionMap</a> and domain rr:TriplesMap.
-The triples map to which <a>fnml:functionValue</a> refers to should be an [[RML]] conforming triples map
-that generates an <a>execution</a> description.
+### fnml:FunctionTriplesMap
+
+<dfn>fnml:FunctionTriplesMap</dfn> is a subclass of [rr:TripleMap](http://www.w3.org/ns/r2rml#TriplesMap),
+to denote that this [triples map](https://rml.io/specs/rml/#term-map) is also a [=function triples map=].
```

Comment:
This sentence is a bit unclear. I don't think its necessary. The link for triples map points to term map by the way.

Reply:
removed this line 68 and fixed the reference :)